app/views/layouts/megatron/application.html.slim

```diff
@@ -8,10 +8,12 @@ html lang="en"
     meta name="viewport" content="width=device-width,initial-scale=1"
     meta charset="utf-8"
     meta http-equiv="X-UA-Compatible" content="IE=edge"
+    meta name="bugsnag" data-api-key="#{ENV['BUGSNAG_API_KEY']}" data-release-stage="#{Rails.env}"
```

Comment:
Do you folks add the bugsnag API key to your devkit settings locally? That's the only way it would be set in development, and I'm not sure why you'd do that (other than testing).

Either way I think we can move forward with this the way it's written :+1: